### ISSUE TITLE
request: /punish command for easier punishment

### ISSUE THREAD

**Issue by ghost** [DevLeoko/AdvancedBan#81]
I love the time layouts and reason layouts! Absolutely amazing!
But could you add a feature where you can create a "complete" layout with different types of punishments, the times, and the reason? So that after configuration you could simply do /punish <player> <layout> to apply the proper punishment (warn, kick, mute, ban, etc), the proper time, and the reason layout according to what has been configured.

Goals:

1. Make punishing players as easy as possible for the staff so they don't have to look at a list of punishments/times, how many previous offenses they have, and then punish them accordingly. 

2. Create a level of transparency and make it more difficult to punish based on how much you like that person, etc. 

3. Make it a bit easier to configure punishments that anti cheat, chat control, and pvp plugins issue automatically

Thanks!

**Comment by ItzSomebody** [comment]
but wait, isn't that what Layout.yml is for?

**Comment by TortillaSquad** [comment]
Yes and no. As far as i know you create a message layout and a separate time layout. You then have to use /<ban|mute|warn|etc> <player> <time|@TimeLayout> <reason|@Layout> and this executes the specified punishment with the specified time/layout and reason/layout.

**Comment by DevLeoko** [comment]
Ya this is a really grat idea :) I'll definitely add this in one of the next updates

**Comment by TortillaSquad** [comment]
Thanks!
